src/late_multicellular_stage/MulticellularMetaballDisplayer.cs

```diff
@@ -34,7 +34,7 @@ public override void _Ready()
         base._Ready();
 
         // This is here in case we need custom shader effects at some point
-        // Material = new ShaderMaterial()
+        // material = new ShaderMaterial()
         // {
         //     Shader = GD.Load<Shader>("res://shaders/Metaball.shader"),
```

Comment:
I think this needs to reverted as if the code is kept as-is when uncommented it shouldn't compile as Godot properties are capitalized.

Reply:
Right, I just thought that material here referred to the 'material' field of this class

Reply:
I could be mistaken, but if this was previously exactly like `Material = new ...` then I think this is likely a placeholder in case the `Material` property of the current Node needs to use a more advanced shader in the future.